src/gallia/command.py

```diff
@@ -226,9 +256,45 @@ def prepare_artifactsdir(
 
         raise ValueError("base_dir or force_path must be different from None")
 
-    @abstractmethod
-    def run(self, args: Namespace) -> int:
-        ...
+    def entry_point(self, args: Namespace) -> int:
+        if self.ARTIFACTSDIR:
+            self.artifacts_dir = self.prepare_artifactsdir(
+                args.artifacts_base,
+                args.artifacts_dir,
+            )
+            setup_logging(
+                self.get_log_level(args),
+                self.get_file_log_level(args),
+                self.artifacts_dir.joinpath("log.json.zst"),
+            )
+        else:
+            setup_logging(self.get_log_level(args))
+
+        exit_code = 0
+        try:
+            exit_code = self.run(args)
+        except (BrokenPipeError, ConnectionRefusedError) as e:
```

Comment:
@peckto Should we make this list configurable and provide these two as a default within gallia?